zebra-utils/src/bin/zebra-checkpoints/args.rs

```diff
@@ -2,11 +2,19 @@ use structopt::StructOpt;
 
 #[derive(Debug, StructOpt)]
 pub struct Args {
+    /// Path to zcash-cli command
+    #[structopt(short, long)]
+    pub cli: String,
+
+    /// Use the mainnet network
+    #[structopt(short, long)]
+    pub mainnet: bool,
+
+    /// Use the regtest network
+    #[structopt(short, long)]
+    pub regtest: bool,
+
     /// Use the test network
     #[structopt(short, long)]
     pub testnet: bool,
```

Comment:
Are these options mutually exclusive? If so it might be better to use an enum and combine them into one option.

Reply:
from below it looks like they are mutually exclusive and that `mainnet` should be the default

Reply:
You should be able to use `structopt::clap::arg_enum` to do this btw

https://docs.rs/clap/2.33.1/clap/macro.arg_enum.html

Reply:
We have a network enum already: `zebra_chain::Network`.

Whatever we do here, we should do the same for the zebrad arguments.

Reply:
Why do you need these arguments?

The original scripts just pass them through.

Reply:
I tried @yaahc `arg_enum` at https://github.com/ZcashFoundation/zebra/pull/740/commits/fffeac01063c114d7219e1377fde703a611f2791 before seeing the comments from @teor2345 . I actually implemented the passthrough at https://github.com/ZcashFoundation/zebra/pull/740/commits/4655f8192cfe1b6d8cc24bf9da15ae25a1180005